.github/workflows/patch.yml

```diff
@@ -0,0 +1,50 @@
+name: Patch
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [main]
+
+jobs:
+  integration-table-gen:
+    needs: [docs]
```

Comment:
This is an insane nitpick, sorry in advance lol, but can you move `integration-table-gen` underneath `docs` to also make the order similar to the `needs`